wagtail_wordpress_import/importers/wordpress.py

```diff
@@ -306,7 +306,7 @@ def connect_page_categories(self, page, category_model, item):
             categories = [
                 category
```

Comment:
**Question**: Our example category model has `name = CharField` https://github.com/torchbox/wagtail-wordpress-import/blob/d158ce198a2cc785a8f37591b0a59369d2fc735c/docs/categories.md#example-category-model.

Given that, category names will be coerce to a string anyway when they're saved, but is it worth calling `str()` on this line too, e.g. to help with deduplication or consistency?

Reply:
Thanks Nick great idea, done.